render/detailed/connections.go

```diff
@@ -115,7 +110,7 @@ func incomingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 		TopologyID:  topologyID,
 		Label:       "Inbound",
 		Columns:     columnHeaders,
-		Connections: connectionRows(r, counts, isInternetNode(n)),
+		Connections: connectionRows(r, ns, counts, isInternetNode(n)),
```

Comment:
The former yes. The latter, no. As a result we fail to look up the reverse DNS resolution in `connectionRows()`. Grrr.